srcpkgs/nvidia/update

```diff
@@ -1,2 +1,2 @@
 site="http://www.nvidia.com/object/unix.html"
-pattern='Long Lived.+>\K[\d.]+(?=</A>)'
+pattern='Short Lived.+>\K[\d.]+(?=</A>)'
```

Comment:
Unless the page also lists beta drivers sometimes (it currently doesn't) I would also leave the branch name out of the update check. Otherwise it would have to be switched back and forth when the package changes the branch.

Reply:
Changed to 'Branch Version'. Should be more robust, 'Latest' is another alternative but would probably pick up beta drivers if they are on the page.